src/main/java/elements/Button.java

```diff
@@ -1,7 +1,8 @@
 package elements;
 
-@ImplementedBy(ButtonImp.class)
+
+@ImplementedBy(ButtonImpl.class)
 public interface Button extends Element {
 
-     void click();
+     void clickBtn();
```

Comment:
why did you rename it? i think that's obvious when we call 'click' method from a button that it related to button

Reply:
renamed to click